MAINTAINERS.md

```diff
@@ -1,22 +1,22 @@
-| Rohit Aggarwal   | [rohitagg2020](https://github.com/rohitagg2020)       | [VMware](https://www.github.com/vmware/) | [kapp](https://github.com/vmware-tanzu/carvel-kapp) (Reviewer), [kctrl](https://github.com/vmware-tanzu/carvel-kapp-controller/tree/develop/cli) (Reviewer)                                                                                                                                                                                |
-| John Brunton     | [jbrunton](https://github.com/jbrunton)               | [converge.io](https://converge.io/)      | [carvel-setup-action](https://github.com/vmware-tanzu/carvel-setup-action) (Approver)                                                                                                                                              |
-| Kumari Tanushree | [kumaritanushree](https://github.com/kumaritanushree) | [VMware](https://www.github.com/vmware/) | [kapp](https://github.com/vmware-tanzu/carvel-kapp)                                                                                                                                                                                |
+| João Pereira     | [joaopapereira](https://github.com/joaopapereira)     | [VMware](https://www.github.com/vmware/) | [imgpkg](https://github.com/carvel-dev/imgpkg) (Lead), [kbld](https://github.com/carvel-dev/kbld), [vendir](https://github.com/carvel-dev/vendir) |
+| John Brunton     | [jbrunton](https://github.com/jbrunton)               | [converge.io](https://converge.io/)      | [carvel-setup-action](https://github.com/carvel-dev/setup-action) (Approver) |
+| Kumari Tanushree | [kumaritanushree](https://github.com/kumaritanushree) | [VMware](https://www.github.com/vmware/) | [vendir](https://github.com/carvel-dev/vendir), [kapp](https://github.com/carvel-dev/kapp) |
+| Neil Hickey      | [neil-hickey](https://github.com/neil-hickey)         | [VMware](https://www.github.com/vmware/) | [kapp-controller](https://github.com/carvel-dev/kapp-controller) (Reviewer), [secretgen-controller](https://github.com/carvel-dev/secretgen-controller) |
+| Praveen Rewar    | [praveenrewar](https://github.com/praveenrewar)       | [VMware](https://www.github.com/vmware/) | [kapp](https://github.com/carvel-dev/kapp) (Approver), [kctrl](https://github.com/carvel-dev/kapp-controller/tree/develop/cli) (Approver), [kapp-controller](https://github.com/carvel-dev/kapp-controller), [imgpkg](https://github.com/carvel-dev/imgpkg) |
+| Rohit Aggarwal   | [rohitagg2020](https://github.com/rohitagg2020)       | [VMware](https://www.github.com/vmware/) | [kapp](https://github.com/carvel-dev/kapp) (Reviewer), [kctrl](https://github.com/carvel-dev/kapp-controller/tree/develop/cli) (Reviewer), [kapp-controller](https://github.com/carvel-dev/kapp-controller) (Reviewer), [secretgen-controller](https://github.com/carvel-dev/secretgen-controller) |
```

Comment:
`[kapp-controller](https://github.com/carvel-dev/kapp-controller) (Reviewer)` I am guessing that this was unintended? (Happy to sponsor Rohit for becoming a reviewer though 😄 )